streamer/src/nonblocking/quic.rs

```diff
@@ -1693,6 +1704,15 @@ pub mod test {
         assert_eq!(stats.connection_remove_failed.load(Ordering::Relaxed), 0);
     }
 
+    pub fn tpu_max_unstaked_streams_per_100ms() -> u64 {
```

Comment:
didn't want to use this for tpu.rs as well?

Reply:
won't be exposed to external crates 'cause `self::test::*` are `cfg(test)`, right?

Reply:
you would have to move it out of `mod test`, yes.

Reply:
but basically have a function in streamer that you use in tests and also core consumes because it seems like we need it in both places